modules/backup-and-restore/pages/configurations.adoc

```diff
@@ -146,9 +152,46 @@ gadmin config set "System.Backup.S3.AWSSecretAccessKey" "<secret-access-key>"
 ----
 gadmin config apply -y
 ----
+
+== Backup to Azure Blob
+
+Similar to AWS S3, once the Azure Blob Container is created and configured properly, for more information,see https://learn.microsoft.com/en-us/azure/storage/blobs/storage-blobs-introduction[Introduction to Azure Blob Storage],
+users only need to complete the configurations by the following steps:
+
+. Enable storing backup data in S3
++
+[console,]
+----
+gadmin config set "System.Backup.S3.Enable" "true"
+----
+. Specify the backup S3 Endpoint
++
+[console,]
+----
+gadmin config set "System.Backup.S3.Endpoint" "https://<account-name>.blob.core.windows.net"
+----
+. Specify bucket name
++
+[console,]
+----
+gadmin config set "System.Backup.S3.BucketName" "<container-name>"
+----
+. Set S3 backup AccessKeyID
++
+[console,]
+----
+gadmin config set "System.Backup.S3.AWSAccessKeyID" "<account-name>"
+----
+
+. Set S3 backup SecretAccessKey
++
+[console,]
+----
+gadmin config set "System.Backup.S3.AWSSecretAccessKey" "<account-key>"
+----
+. Apply the new parameter values
 +
-.Restart all services
```

Comment:
It is unnecessary to restart all services if we only change the backup configurations. Running `gadmin config apply` is enough to make it effective.